script/prerender.tsx

```diff
@@ -5,7 +5,6 @@ import path from 'path';
 import portfinder from 'portfinder';
 import React from 'react';
 import { renderToString } from 'react-dom/server';
-import { ServerStyleSheet, StyleSheetManager } from 'styled-components';
```

Comment:
without this stuff in pre-render, how do the styles get into the pre-rendered html?

Reply:
Ahh, maybe I need to include the `<CacheProvider>` (from https://emotion.sh/docs/ssr ) . I will try later

Reply:
oh apparently you don't need anything special for ssr in emotion because it injects the style tags adjacent to the elements